### PR TITLE
Extend marshaller to patch translations

### PR DESCRIPTION
**Pull request by berarma**
I've implemented this quick solution so that I could patch all translations in one operation.

### CODE REVIEW

File: src/ORM/Marshaller.php [hunk 1]
```diff
@@ -479,6 +481,16 @@ public function merge(EntityInterface $entity, array $data, array $options = [])
         return $entity;
     }
 
+    public function mergeTranslations($original, $value) {
```
Comment (markstory) [inline src/ORM/Marshaller.php]: Functions need comments. Does this method really need to be public?

File: src/ORM/Marshaller.php [hunk 2]
```diff
@@ -479,6 +481,16 @@ public function merge(EntityInterface $entity, array $data, array $options = [])
...
+        foreach ($value as $language => $fields) {
+            if (!isset($original[$language])) {
+                $original[$language] = new Entity();
```
Comment (markstory) [inline src/ORM/Marshaller.php]: What if the association entity is not the default class?

File: src/ORM/Marshaller.php (hunk 1 above)
Comment (ADmad) [inline src/ORM/Marshaller.php]: Should translation specific code be in marshaller?

Comment (markstory) [inline src/ORM/Marshaller.php]: I'm not super excited about it, but I can't think of a better solution right now.

**Comment by lorenzo** [comment]
I like this idea of having this, but the implementation needs to change, either by finding a cleaner way or by improving the core so that custom marshallers can be used. I'm closing this PR since it has been inactive for a long time, we can revisit having this feature at a later point

**Pull request closed without merging**